assets/src/application/ui/input/Text.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { Input } from 'antd';
+import { InputProps } from 'antd/lib/input';
 
-const Text = ({ input, htmlType = 'text', ...rest }) => {
-	return <Input {...input} {...rest} type={htmlType} />;
+const Text: React.FC<InputProps> = ({ type = 'text', ...props }) => {
```

Comment:
Because `InputProps` doesn't have `htmlType`, I've tracked down why is that and found next:
- `InputProps` extends `Omit<React.InputHTMLAttributes<HTMLInputElement>, 'size' | 'prefix'>`: https://github.com/ant-design/ant-design/blob/master/components/input/Input.tsx#L15
which has this type: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2057
- In `Button` component antd uses `htmlType` although which was a bit confusing: https://github.com/ant-design/ant-design/blob/master/components/button/button.tsx#L100